gateway/cli/src/main.rs

```diff
@@ -181,7 +181,12 @@ pub fn source_password(rpcpassword: Option<String>) -> String {
     match rpcpassword {
         None => rpassword::prompt_password("Enter gateway password:").unwrap(),
         Some(password) => {
-            println!("WARNING: Passing in a password from the command line may be less secure!");
+            /*
+            TODO: Warn users against passing in passwords from the command line
+            This is currently disabled because we need clean json results from the cli
+            when running intergretaion tests based on shell scripts
+            */
+            // println!("WARNING: Passing in a password from the command line may be less secure!");
```

Comment:
Can you use `eprintln!` instead? That shouldn't be piped to `jq` in the tests.

Reply:
Oh that works! Thanks :)